src/chart/onResize.js

```diff
@@ -164,15 +173,17 @@ export default function onResize() {
             this.config.formDescription_col
           ]
       );
-  if (this.config.y.column === "Field" && this.config.fieldDescription_col)
+  if (this.config.y.column === "Form: Field" && this.config.fieldDescription_col)
```

Comment:
Actually - not getting titles in `v1.0.0-dev` branch either, so probably an issue I introduced with the merge.